index.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="styles.css">
+    <title>Kopie rekupe</title>
+</head>
+<body>
+<a href="https://www.rekupe.com/" target="_blank">Rekupe pagina</a>
+<header class="outer-container">
+    <nav class="navbar">
+        <div >
+            <ul>
+                <!--Let op hier nog verwijzen naar betreffende onderdeel-->
+                <li><a href="#casestudies">Case studies</a></li>
+                <li><a href="#services">Services</a></li>
+                <li><a href="#contactus">Contact Us</a></li>
+            </ul>
+        </div>
+    </nav>
+
+
+        <div class="outer-hero">
+            <div class="hero-section">
+                <h1>Your website is an investment in the success of your business.</h1>
+                <p>We are here to help you succeed.</p>
+                <button type="button" class="cta-btn">Let's work together</button>
+            </div>
+                <object class="sqline" data="assets/squiggle_line_stripe.svg"></object>
```

Comment:
Je hebt een object class gemaakt. Wanneer het gaat om een image gebruiken we uitsluitend de <  img > tag. Tevens kon je hier ook een < div > van maken en de image instellen als achtergrond! (Dit niet voor alle images doen, eerder alleen voor icons e.d.)